tests/unit_tests/test_wave_06.py

```diff
@@ -227,8 +236,16 @@ def test_swap_best_by_category_no_match_is_false():
         my_priority="Clothing",
         their_priority="Clothing"
     )
-
-    raise Exception("Complete this test according to comments below.")
+    assert result is False
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert item_a in tai.inventory
+    assert item_b in tai.inventory
+    assert item_c in tai.inventory
+    assert item_d in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_f in jesse.inventory
```

Comment:
Yeah, like this. 😄  Except, careful with copying and pasting–there are two `item_d` cases.